doc/source/guidelines/doc_practices.rst

```diff
@@ -0,0 +1,352 @@
+.. _doc_practices:
+
+Documentation Practices
+=======================
+The generation of documentation uses `Sphinx <https://www.sphinx-doc.org/en/master/>`__
+and an Ansys-branded theme. Documentation is built from docstrings, reStructuredText
+(RST) files, and Python (PY) example files.
+
+Docstrings
+----------
+Docstrings must be formatted so that Sphinx can parse them. You can use either the 
+``numpydoc``or ``napoleon`` Sphinx extension:
+ 
+ - `numpydoc <https://pypi.org/project/numpydoc/>`_
+ - `napoleon <https://pypi.org/project/sphinxcontrib-napoleon/>`_
+
+The ``numpydoc`` extension is generally preferred because it supports an API
+documentation structure with one page per method, providing Python community
+members with documentation like that generated by `pandas <https://pandas.pydata.org/>`_
+and `NumPy <https://numpy.org/>`_. If, however, your API is very linear, you
+might prefer the  ``napoleon`` extension because it supports a documentation
+structure where everything needed to solve a certain problem can be shown on one page.
+
+The ``numpydoc`` extension provides its own `style guide <https://numpydoc.readthedocs.io/en/latest/format.html>`_
+and a `user guide <https://numpydoc.readthedocs.io/en/latest/>`_ that explains how to use
+the extension with Sphinx. The ``napoleon`` extension, which parses both numpy and Google
+style docstrings, refers you to the `Google Python Style Guide <https://google.github.io/styleguide/pyguide.html>`_.
```

Comment:
I think we should suggest a consistent docstring format, regardless of whether `numpydoc` or `napoleon` is used to generate the documentation.

Since `napoleon` supports both NumPy-style and Google-stye, I would suggest recommending NumPy-style in both cases.